doc/releases/changelog-0.9.0.md

```diff
@@ -166,204 +180,221 @@
 
   Using operator assignment syntax in favor of `at...op` expressions is now possible for the
   following operations:
-  * `x[i] += y` in favor of `x.at[i].add(y)`
-  * `x[i] -= y` in favor of `x.at[i].add(-y)`
-  * `x[i] *= y` in favor of `x.at[i].multiply(y)`
-  * `x[i] /= y` in favor of `x.at[i].divide(y)`
-  * `x[i] **= y` in favor of `x.at[i].power(y)`
+
+  - `x[i] += y` in favor of `x.at[i].add(y)`
+  - `x[i] -= y` in favor of `x.at[i].add(-y)`
+  - `x[i] *= y` in favor of `x.at[i].multiply(y)`
+  - `x[i] /= y` in favor of `x.at[i].divide(y)`
+  - `x[i] **= y` in favor of `x.at[i].power(y)`
 
   ```python
   @qjit(autograph=True)
   def f(x):
-    first_dim = x.shape[0]
-    result = jnp.copy(x)
+      first_dim = x.shape[0]
+      result = jnp.copy(x)
 
-    for i in range(first_dim):
-      result[i] *= 2  # This is now supported
+      for i in range(first_dim):
+        result[i] *= 2  # This is now supported
 
-    return result
+      return result
   ```
 
   ```pycon
   >>> f(jnp.array([1, 2, 3]))
   Array([2, 4, 6], dtype=int64)
   ```
 
-* Catalyst now has a standalone compiler tool called `catalyst-cli` which quantum
-  compiles MLIR input files into an object file without any dependancy to the python frontend.
+* Catalyst now has a standalone compiler tool called `catalyst-cli` that quantum compiles MLIR input
+  files into an object file independent of the Python frontend.
```

Comment:
From Josh (https://github.com/PennyLaneAI/catalyst/pull/1239#discussion_r1820351702):

> @mehrdad2m I noticed that there is no documentation for catalyst-cli aside from the changelog entry, which will likely make it hard for developers to use/us to maintain it.

> Likely the best approach here is to add a catalyst-cli page to the API section of the docs:

> with the usage/documentation of the CLI tool pulled directly from the source code/--help output, to make it more maintainable (does catalyst-cli --help work?)

Reply:
We'll follow-up in #1249.